lib/reversetunnel/srv.go

```diff
@@ -634,6 +634,7 @@ func (s *server) handleTransport(sconn *ssh.ServerConn, nch ssh.NewChannel) {
 		component:        teleport.ComponentReverseTunnelServer,
 		localClusterName: s.ClusterName,
 		emitter:          s.Emitter,
+		sconn:            sconn,
```

Comment:
I'm a little unsure if injecting this connection into here has some other meanings or repercussions that I don't know about.